src/types/models/BalancedGrowthCommunityModel.jl

```diff
@@ -0,0 +1,287 @@
+"""
+$(TYPEDEF)
+
+A standardized structure used to package models that can easily be combined into
+a [`BalancedGrowthCommunityModel`](@ref).
+
+# Fields
+$(TYPEDFIELDS)
+
+# Assumptions
+1. Exchange reactions, *all* of which are idenitified in `exchange_reaction_ids`,
+   have the form: `A[external] ⟷ ∅` where `A` is a metabolite. No other
+   exchanges are allowed.
+2. The biomass reaction of a model produces a biomass metabolite called
+   `biomass_metabolite_id` with stoichiometry 1.
+3. There is only one biomass reaction in the model.
+"""
+Base.@kwdef mutable struct CommunityMember
+    "Name of model appended to intracellular reactions and metabolites."
+    id::String
+    "Abundance of organism."
+    abundance::Float64
+    "Underlying model."
+    model::AbstractMetabolicModel
+    "List of all exchange reactions in model."
+    exchange_reaction_ids::Vector{String}
+    "ID of biomass metabolite."
+    biomass_metabolite_id::String
```

Comment:
TODO for myself later: biomass balancing is a function of the class that provides the balance, so the metabolite (= equality constraint) should be created externally, not by the community member.